ansible_collections/arista/avd/molecule/cv_deploy/workspace_force_true.yml

```
@@ -0,0 +1,61 @@
+---
+- name: Converge - cv_deploy with cv_submit_workspace_force = true
+  hosts: SITE1_FABRIC
+  connection: local
+  gather_facts: false
+  vars:
+    cv_server: www.cv-staging.corp.arista.io
+    cv_token: "{{ lookup('env', 'CVAAS_AAWG_CI') }}"
+    cv_verify_certs: false
+    cv_skip_missing_devices: true
+    eos_config_dir: "{{ playbook_dir }}/intended/configs/test_configs"
+    structured_dir: "{{ playbook_dir }}/intended/structured_configs/test_configs"
+    intended_tag_device: avd-ci-leaf1
+    intended_tags: "{{ lookup('file', structured_dir ~ '/' ~ intended_tag_device ~ '.yml')| from_yaml }}"
+
+  tasks:
+    - name: Generate random string
+      ansible.builtin.set_fact:
+        r: "{{ lookup('password', '/dev/null chars=ascii_lowercase,digits length=4') }}"
+      run_once: true
+
+    - name: Set facts
+      ansible.builtin.set_fact:
+        cv_workspace_name: avd-cv-deploy-{{ r }}
+        cv_workspace_description: sample description
+        cv_change_control_name: cc_cv_deploy-{{ r }}
+        cv_change_control_description: sample description
+        cv_register_detailed_results: true
+      run_once: true
+
+    - name: Provision with cv_submit_workspace_force = true
+      run_once: true
+      delegate_to: localhost
+      ansible.builtin.import_role:
+        name: arista.avd.cv_deploy
+      vars:
+        cv_devices: [ avd-ci-leaf1, avd-ci-core1 ]
+        cv_submit_workspace_force: true
+
+    - name: Display CVP result
+      run_once: true
+      ansible.builtin.debug:
+        msg: '{{ cv_deploy_results }}'
+
+    - name: Check CVP returns
+      ansible.builtin.assert:
+        that:
+          # errors and warnings
+          - cv_deploy_results.errors == []
```

The following call can be used to clean/delete CC created by this task but not approved/executed (due to the nature of the test). This will help to keep UI/Aeris clean
```suggestion
          - cv_deploy_results.errors == []

    - name: Cleanup orphan CC
      run_once: true
      uri:
        url: https://{{ cv_server }}/api/resources/changecontrol/v1/ChangeControlConfig?key.id={{ cv_deploy_results.change_control.id }}
        validate_certs: yes
        return_content: yes
        headers:
          Accept: "application/json"
          Content-Type: "application/json"
          Authorization: "Bearer {{ lookup('env', 'CVAAS_AAWG_CI') }}"
        method: DELETE
        body_format: json
        force_basic_auth: yes
        timeout: 10
      register: cvp_abandon_cc_result
      until: cvp_abandon_cc_result.status == 200
      retries: 3
      delay: 3
      ignore_errors: True
      when: cv_deploy_results.change_control.state | lower in ['pending approval', 'approved']
```